crispy_bootstrap5/bootstrap5.py

```diff
@@ -7,3 +8,44 @@ class FloatingField(Field):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.attrs["placeholder"] = self.fields[0]
+
+
+class BS5Accordion(Accordion):
+    """
+    Bootstrap5 Accordion menu object. It wraps `BS5AccordionGroup` objects in a
+    container. It also allows the usage of accordion-flush, introduced in bootstrap5::
+
+        BS5Accordion(
+            BS5AccordionGroup("group name", "form_field_1", "form_field_2"),
+            BS5AccordionGroup("another group name", "form_field"),
+            flush=True,
+            always_open=True
+        )
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.flush = kwargs.pop("flush", False)
+        self.always_open = kwargs.pop("always_open", False)
+
+        if self.always_open:
+            for accordion_group in args:
```

Comment:
`self.fields` works. I'll pathc it

Would you shed some light on this if you can? In the original Accordion class both are used, `args` in the `super().__init__()` and `self.fields` in the render() function. I went with `args` because I'm also writing code in the `super().__init__()` method.

What is the difference? Both seem to do the same thing for me 🤔

Reply:
They are both a list of fields (or in this case accorion groups) within the accordion. I just thought it is more readable to understand when written as `self.fields` rather than an unspecific `args`.

Reply:
Thanks for explaining! I think you're right, `self.fields` seems more readable to me as well.